src/NearOrg/Notifications/NotificationButton.jsx

```diff
@@ -157,11 +163,15 @@ const Notification = ({ count, disabled }) => {
                 src="${REPL_ACCOUNT}/widget/DIG.Button"
                 props={{
                   href: "/notifications",
-                  fill: "outline",
                   variant: "secondary",
```

Comment:
```suggestion
                  variant: "primary",
                  fill: "outline",
```

Reply:
This has been addressed in the latest commits.